readme.md

```diff
@@ -109,6 +109,38 @@ await throttled();
 //=> Executing...
 ```
 
+##### signal
+
+Type: [`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal)
+
+An optional signal to listen for abort events. If the signal becomes aborted, all pending and future calls are rejected with a `pThrottle.AbortError` error.
```

Comment:
This is not the correct behavior. It should reject with [AbortSignal.reason](dfile:///Users/sindresorhus/Library/Application%20Support/Dash/DocSets/JavaScript/JavaScript.docset/Contents/Resources/Documents/developer.mozilla.org/en-US/docs/Web/API/AbortSignal/reason.html)